rest/tests/base.py

```diff
@@ -0,0 +1,389 @@
+"""
+LORE test case
+"""
+
+from __future__ import unicode_literals
+import json
+
+from rest_framework.status import (
+    HTTP_200_OK,
+    HTTP_201_CREATED,
+    HTTP_204_NO_CONTENT,
+)
+from rest_framework.reverse import reverse
+
+from learningresources.tests.base import LoreTestCase
+
+API_BASE = '/api/v1/'
+REPO_BASE = '/api/v1/repositories/'
+
+
+def as_json(resp):
+    """Get JSON from response"""
+    return json.loads(resp.content.decode('utf-8'))
+
+
+# pylint: disable=too-many-public-methods, too-many-arguments
+class RESTTestCase(LoreTestCase):
+    """
+    REST test case
+    """
+
+    # pylint: disable=dangerous-default-value
+    # Do not mutate these fields!
+    DEFAULT_REPO_DICT = {
+        'name': 'name',
+        'description': 'description',
+    }
+    DEFAULT_VOCAB_DICT = {
+        'name': 'name',
+        'description': 'description',
+        'required': True,
+        'weight': 1000,
+        'vocabulary_type': 'f',
+    }
+    DEFAULT_TERM_DICT = {
+        'label': 'term label',
+        'weight': 1000,
+    }
+
+    def assert_options_head(self, url, expected_status):
+        """Assert OPTIONS and HEAD"""
+        resp = self.client.options(url)
+        self.assertEqual(expected_status, resp.status_code)
+        as_json(resp)
+        self.assertIn("HEAD", resp['ALLOW'])
+        self.assertIn("OPTIONS", resp['ALLOW'])
```

Comment:
Okay. It's really a nit, anyway. If the server was misconfigured in a way that responded to `OPTIONS` while disabling `OPTIONS` in the header then this would find it, and it definitely doesn't hurt anything.